packages/admin/admin/src/dataGrid/muiGridFilterToGql.tsx

```diff
@@ -61,14 +61,14 @@ function convertValueByType(value: string, type?: string) {
 export function muiGridFilterToGql(columns: GridColDef[], filterModel?: GridFilterModel): { filter: GqlFilter; search?: string } {
     if (!filterModel) return { filter: {} };
     const filterItems = filterModel.items
-        .filter((value) => value.value !== undefined)
-        .map((value) => {
-            if (!value.operatorValue) throw new Error("operaturValue not set");
-            const gqlOperator = muiGridOperatorValueToGqlOperator[value.operatorValue] || value.operatorValue;
-            const column = columns.find((i) => i.field == value.columnField);
-            const convertedValue = convertValueByType(value.value, column?.type);
+        .filter((filterItem) => filterItem.value !== undefined)
+        .map((filterItem) => {
+            if (!filterItem.operatorValue) throw new Error("operaturValue not set");
+            const gqlOperator = muiGridOperatorValueToGqlOperator[filterItem.operatorValue];
```

Comment:
please review your own changes carefully

in this line you are not just renaming value but also removing code